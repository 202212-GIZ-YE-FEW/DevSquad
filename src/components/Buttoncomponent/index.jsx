import React from "react";

export default function Buttoncomponent({
    name = " ",
    bgColor = "slate-300",
    textColor = "text-white",
    width,
    height,
    font = "font-Rubik",
    fontSize,
    borderColor,
    borderRaduis,
    border,
    margin,
    padding,
}) {
    return (
        <button
            className={`${bgColor} ${textColor} ${width} ${height} ${font} ${fontSize} ${borderColor} ${borderRaduis} ${border} ${margin} ${padding}`}
        >
            {name}
        </button>
    );
}
