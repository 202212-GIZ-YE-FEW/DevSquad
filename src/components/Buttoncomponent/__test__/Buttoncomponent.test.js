import renderer from "react-test-renderer";

import Buttoncomponent from "../index";

it("renders correctly", () => {
    const tree = renderer.create(<Buttoncomponent />).toJSON();
    expect(tree).toMatchSnapshot();
});
