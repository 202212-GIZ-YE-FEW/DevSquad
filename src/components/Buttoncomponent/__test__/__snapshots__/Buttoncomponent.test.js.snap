// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="slate-300 text-black undefined undefined font-Rubik    undefined m-2 p-2"
>
   
</button>
`;
